release-notes/SupportedFeatures-v1.0.0.md

```diff
@@ -34,23 +34,22 @@ Supported features
 |Service Authentication|Certificate    |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
-|              |SPNIdentity/HTTP       |:white_check_mark:                   |:white_check_mark:                  |[:warning:](#known_issues)                 | [:warning:](#known_issues)          |
-|              |SPNIdentity/TCP        |[:warning:](#known_issues)           |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
-|              |UPNIdentity    |[:warning:](#known_issues)          |[:warning:](#known_issues)          |[:warning:](#known_issues)                  | [:warning:](#known_issues)          |
+|              |SPNIdentity    |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)                  | [:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |
+|              |UPNIdentity    |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)                  | [:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |
 |Client types  |ChannelFactory\<T\>    |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
 |              |ChannelBase\<T\>       |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
 |Extensibility |IClientMessageInspector|:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
 |              |IClientMessageFormatter|:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
-|Management    |ETW Tracing            |[:no_entry_sign:](#known_issues)      |:white_check_mark:                  |[:no_entry_sign:](#known_issues)          |[:no_entry_sign:](#known_issues)     |          
+|Management    |ETW Tracing            |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)      |:white_check_mark:                  |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)     |          
 
 
 #### Platforms
```

Comment:
NIT: line 58, "... include..."
Also can we update Linux distro names to match the names in dotnet/core#142 (it's okay to skip versions but match the full name and casing)?